lib/widgets/matrix.dart

```diff
@@ -757,6 +764,24 @@ class MatrixState extends State<Matrix>
     didChangeAppLifecycleState(AppLifecycleState.paused);
   }
 
+  Future<void> _setupAuthUrl({
+    String? url,
+  }) async {
+    final newAuthUrl = loginHomeserverSummary?.discoveryInformation
```

Comment:
IMO this can be moved inside the `else` braces since `if (url != null)` `newAuthUrl` is not used

Reply:
Sure